clusterloader2/testing/load/deployment.yaml

```diff
@@ -51,6 +51,11 @@ spec:
         - name: configmap
           configMap:
             name: {{.BaseName}}-{{.Index}}
+{{if false}}
```

Comment:
Do we need to mount only selected items? I think it makes sense to mount all and using only /var/configmap/feature-name if needed.

Reply:
I wanted to have minimal changes, but yeah - we can go with the eventual solution from the beginning.